test/integration_test/specs/kubevirt-fedora-vm/fedora-vm.yaml

```diff
@@ -0,0 +1,67 @@
+apiVersion: kubevirt.io/v1
+kind: VirtualMachine
+metadata:
+  annotations:
+    kubevirt.io/latest-observed-api-version: v1
+    kubevirt.io/storage-observed-api-version: v1alpha3
```

Comment:
remove the annotations since they will be added by the kubevirt?

same for "generation"?

Reply:
Fixed.